src/lib.rs

```diff
@@ -44,47 +43,11 @@ pub mod runtest;
 pub mod common;
 pub mod errors;
 
+pub use common::Config;
+
+#[deprecated(since="0.2.9", note="Use Config::default() instead")]
```

Comment:
Loving this! Maybe include something like: `This method will be removed in version 0.3.0`